### PR TITLE
Add new manual source

### PR DESCRIPTION
**Pull request by adamnsch**

### CODE REVIEW

**Comment by adamnsch** [comment]
Currently getting the following error when trying `npm run start`:
```bash
npm run start

> graph-data-science@2.2.0-preview start
> nodemon -e adoc --exec "npm run build && npm run serve"

[nodemon] 2.0.19
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: adoc
[nodemon] starting `npm run build && npm run serve`

> graph-data-science@2.2.0-preview build
> antora preview.yml --stacktrace --log-format=pretty

[16:31:51.782] FATAL (antora): Local content source must be a git repository: /home/adam/Projects/ (url: ../../)
    Cause: Error
        at loadRepository (/home/adam/Projects/gdsclient-fork/doc/node_modules/@antora/content-aggregator/lib/aggregate-content.js:231:13)
        at async Promise.all (index 0)
        at async Promise.all (index 0)
        at async generateSite (/home/adam/Projects/gdsclient-fork/doc/node_modules/@antora/site-generator/lib/generate-site.js:17:5)
        at async Command.parseAsync (/home/adam/Projects/gdsclient-fork/doc/node_modules/commander/lib/command.js:923:5)
[nodemon] app crashed - waiting for file changes before starting...
```

File: doc/preview.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  url: https://neo4j.com/docs
+  start_page: graph-data-science:ROOT:index.adoc
+
+content:
+  sources:
+  - url: ../../
+    start_path: public/doc
+    branches: 
+    - HEAD
+    include: public/doc/
```
Comment (recrwplay) [inline doc/preview.yml:12]:
```suggestion
  - url: ../
    start_path: doc
    branches: 
    - HEAD
    include: doc/
```

These changes should fix the error you are having when running `npm run start`

File: doc/preview.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  - "@neo4j-antora/antora-table-footnotes"
+  attributes:
+    page-theme: docs
+    page-type: Docs
+    page-search-type: Docs
+    page-search-site: Reference Docs
+    page-canonical-root: /docs
+    page-pagination: true
+    page-no-canonical: true
+    page-origin-private: true
```
Comment (recrwplay) [inline doc/preview.yml:43]:
```suggestion
    page-origin-private: false
```

As this is a public repo, when we publish the docs, by changing this attribute we can include an `Edit this page` link on each page, which links to the page's location in the repo.

File: doc/preview.yml (hunk 1 above)
Comment (adamnsch) [reply doc/preview.yml:12]: works like a charm, thanks!

File: doc/preview.yml (hunk 2 above)
Comment (adamnsch) [reply doc/preview.yml:43]: Cool :) done

**Comment by adamnsch** [comment]
> Currently getting the following error when trying `npm run start`:
> 
> ```shell
> npm run start
> 
> > graph-data-science@2.2.0-preview start
> > nodemon -e adoc --exec "npm run build && npm run serve"
> 
> [nodemon] 2.0.19
> [nodemon] to restart at any time, enter `rs`
> [nodemon] watching path(s): *.*
> [nodemon] watching extensions: adoc
> [nodemon] starting `npm run build && npm run serve`
> 
> > graph-data-science@2.2.0-preview build
> > antora preview.yml --stacktrace --log-format=pretty
> 
> [16:31:51.782] FATAL (antora): Local content source must be a git repository: /home/adam/Projects/ (url: ../../)
>     Cause: Error
>         at loadRepository (/home/adam/Projects/gdsclient-fork/doc/node_modules/@antora/content-aggregator/lib/aggregate-content.js:231:13)
>         at async Promise.all (index 0)
>         at async Promise.all (index 0)
>         at async generateSite (/home/adam/Projects/gdsclient-fork/doc/node_modules/@antora/site-generator/lib/generate-site.js:17:5)
>         at async Command.parseAsync (/home/adam/Projects/gdsclient-fork/doc/node_modules/commander/lib/command.js:923:5)
> [nodemon] app crashed - waiting for file changes before starting...
> ```

this is fixed now

**Comment by adamnsch** [comment]
@recrwplay I've tried updating the configs etc. Wdyt? I'm thinking this is ready for review soon

File: doc/preview.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    url: https://s3-eu-west-1.amazonaws.com/static-content.neo4j.com/build/ui-bundle-latest.zip
+    snapshot: true
+  output_dir: /assets
+
+runtime:
+  fetch: true
+
+urls:
+  html_extension_style: indexify
+
```
Comment (nvitucci) [inline doc/preview.yml:28]: @recrwplay Do we need to add the antora-modify-sitemaps extension here?

```
antora:
  extensions:
  - require: "@neo4j-antora/antora-modify-sitemaps"
    sitemap_version: '1.2.0'
    sitemap_loc_version: 'current'
    move_sitemaps_to_components: true
```

File: doc/publish.yml [hunk 4]
```diff
@@ -0,0 +1,60 @@
+site:
+  title: Neo4j Graph Data Science Client
+  url: https://neo4j.com/docs
+  start_page: graph-data-science-client:ROOT:index.adoc
+
+content:
+  sources:
+  - url: ../../
```
Comment (nvitucci) [inline doc/publish.yml:8]: Should this be `url: ../` too?

File: doc/preview.yml (hunk 3 above)
Comment (nvitucci) [reply doc/preview.yml:28]: (Just realised that this is the preview, so maybe we don't)

File: doc/publish.yml (hunk 4 above)
Comment (adamnsch) [reply doc/publish.yml:8]: That seems right :) ill update it

Comment (adamnsch) [reply doc/publish.yml]: done

Comment (recrwplay) [reply doc/publish.yml]: The paths should be the same as in preview.yml, yes. This playbook will be needed when we want to publish multiple versions of the client docs

File: doc/preview.yml (hunk 3 above)
Comment (recrwplay) [reply doc/preview.yml:28]: In order to publish the sitemap, yes. It can be added now or later.

Comment (adamnsch) [reply doc/preview.yml:28]: ok, done! :)

**Review by nvitucci — APPROVED** [review]
LGTM

**Pull request merged**